data/sql/updates/pending_db_world/rev_1647391921098550500.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1647391921098550500');
+
+-- Update target for quest credit
+UPDATE `smart_scripts` SET `target_type` = 21, `target_param1` = 5 WHERE `entryorguid` = 7207 AND `id` = 1;
```

Comment:
Only 5 yards of max distance? (target_param1)

Reply:
Quest giver to player is very close at the end of waypoint so it should be enough. I tested from 2 to 5. Found 5 will work better if player will lay abit further from quest giver.

Reply:
Hmm but player can move away from the npc. But well, their fault I think :P